test/src/server.py

```diff
@@ -1425,7 +1454,7 @@ def start_vmux(self: 'Host') -> None:
         Returns
         -------
         """
-        self.tmux_new('vmux', f'ulimit -n 4096; sudo {self.vmux_path}')
+        self.tmux_new('vmux', f'ulimit -n 4096; sudo {self.vmux_path}  -d 0000:41:00.0 -s /tmp/vmux-okelmann.sock')
```

Comment:
maybe repurpose vmux_socket_path as vmux_args to make it more flexible against vmux cmdline param changes

Reply:
`vmux_socket_path` is also needed by `Host.run_guest` to run Qemu with the vmux-backed device, so this cannot easily be repurposed. I could add a config parameter for further arguments, but the two already there are kind of critical and probably won't change for now, would they?